packages/autorest.typescript/test/rlcIntegration/generated/bodyStringRest/package.json

```diff
@@ -34,7 +34,7 @@
     "format": "prettier --write --config ../../../.prettierrc.json --ignore-path ../../../.prettierignore \"src/**/*.ts\" \"*.{js,json}\"  \"test/**/*.ts\" \"samples-dev/**/*.ts\"",
     "generate:client": "autorest --typescript swagger/README.md && npm run format",
     "integration-test:browser": "karma start --single-run",
-    "integration-test:node": "nyc mocha -r esm --require source-map-support/register --reporter ../../../common/tools/mocha-multi-reporter.js --timeout 5000000 --full-trace \"dist-esm/test/{,!(browser)/**/}*.spec.js\"",
+    "integration-test:node": "nyc mocha -r esm --require source-map-support/register --timeout 5000000 --full-trace \"dist-esm/test/{,!(browser)/**/}*.spec.js\"",
```

Comment:
I think this command will fail now without the esm-workaround.js in the js-repro.  However, in the old code it also uses `--reporter ../../../common/tools/mocha-multi-reporter.js` which is from js-repo, so it is broken already.

If we really have usage for this, then generating an esm-workaround.js should be simple enough.

Reply:
We would just need to add a generated file and add `-r esm-workaround` to mocha command line options

https://github.com/Azure/azure-sdk-for-js/blob/main/common/tools/esm-workaround.js

Reply:
Thanks for the info and I think we need to fix this in non-branded azure env and tracked here https://github.com/Azure/autorest.typescript/issues/2055

Reply:
Hope we can migrate everything to "type": "module" then workarounds would be no longer needed.